### PR TITLE
Add apps and infra details to technical-details page

### LINKED ISSUE #468: HIPAA technical details page: build out ref arch details
https://github.com/gruntwork-io/gruntwork-io.github.io/pull/456#discussion_r673977607

### LINKED ISSUE #474: HIPAA technical details page: add app and infra "Details" sections
The zig zag portion is a bit short. See [this comment](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/456#discussion_r673986479) for ideas on a few more zigs/zags and more info to put below that.

### PR DESCRIPTION
**Pull request by zackproser**
Closes #468 
Closes #474 

These changes add the Ref Arch details to the Technical Details page: 
* Add zig-zag of infra features 
* Add a modified version of the ref arch features table lifted from the Heroku pricing page

## Copy and markup
- [x] Create page 
- [x] Cross-link across all other pages
- [x] Put copy in place
- [x] Put code snippet viewer in place
- [x] Create a list or table of all the types of infrastructure we support using [this table](https://gruntwork.io/migrate-from-heroku-to-aws/pricing/#compute) as inspiration
- [ ] Finalize copy in the app features table
## Images
Create and add new images for ref-arch detail page - using[ this comment as a guide](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/456#discussion_r673977607):
- [ ] "Customizable" - (image could be a questionnaire asking for EKS vs ECS vs EC2, MySQL vs Postgres vs SQL Server, etc)
- [ ] "Gruntwork deploys it into your AWS accounts and gives you all the code to manage it" - (image could be a pull request with the title "Your architecture has been deployed!" and some links for the customer to try in the body of the PR)

## Bug fixes
- [x] Fix code viewer tab navigation conflicts
- [x] Dry up the Javascript driving the code navigators (currently occurs twice - should be moved out to partial that can be included at bottom of page)
- [x] Implement tab-aware navigation for `/technical-details` page
- [ ]  `/technical-details` page using default nav - should be using hipaa nav

### CODE REVIEW

File: _data/hipaa-apps.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+features:
+  - title: Service discovery
+    description: Apps must look up service endpoints via environment variables, or DNS, or specific IP addresses, or a client-library.
+    available: "yes"
+
```
Comment (brikis98) [inline _data/hipaa-apps.yml:5]: Oh, I didn't mean for this text to be used 100% verbatim. It was a Slack thread and not copywriting meant for a website 😁 

E.g., This should talk about what service discovery is. The "Service Resilience" item below shouldn't say "A ton of other concerns." The logging item says, "Apps that are Dockerized should be able to “just” send," with the "just" responding to a comment from Yori in that Slack thread. 

So all this text needs clean up / polish.

File: _data/hipaa-apps.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - title: Service mesh
+    description: After apps successfully find a service, they also need to authenticate to it, encrypt the communication channel, balance the load, and route traffic in specific ways.
+    available: "yes"
```
Comment (brikis98) [inline _data/hipaa-apps.yml:8]: We don't have service mesh yet.

File: _data/hipaa-apps.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+  - title: Service Resilience
+    description: Apps must handle retries, exponential back-off, jitter, circuit breakers, thread management, async I/O, throttling, error handling, and a ton of other concerns.
+    available: "yes"
+
+  - title: Metrics / observability
+    description: Apps must use encryption by default. Databases, disk volumes, S3 buckets, and machine images must be encrypted. Network connections must be encrypted.
+    available: "yes"
```
Comment (brikis98) [inline _data/hipaa-apps.yml:16]: This description is not for metrics / observability?

File: _data/hipaa-apps.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+  - title: Automated tests
+    description: All maintainable applications need good tests, but setting them up correctly requires knowledge of pre-commit checks, code-formatting, static analysis, unit tests, integration tests, end to end tests, performance tests, testing frameworks, clear test output and more.
+    available: "yes"
+
+  - title: Asset pipeline
+    description: Applications that depend on static content typically require asset compilation (e.g., compile TypeScript to JS), minification, concatenation, fingerprinting, deployment to one or more CDNs, and making the URLs work in dev & prod.
+
+    available: "yes"
```
Comment (brikis98) [inline _data/hipaa-apps.yml:53]: I don't think our sample app does any asset pipeline stuff yet

File: _data/hipaa-apps.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+  - title: Security baseline
+    description: Every app needs a baseline of security features, such as CSRF protection, XSS protection (especially in the application's templating language), security headers, OS user you run as, authn / authz filter, and so on.
+    available: "yes"
```
Comment (brikis98) [inline _data/hipaa-apps.yml:57]: I don't think our sample app does any security headers yet

File: pages/landing/hipaa/technical-details/index.html [hunk 6]
```diff
@@ -69,4 +304,16 @@
...
+  </div>
+  <div id="how-it-works" class="section section-dark">
+    <div class="container">{% include_relative _infra-how-it-works.html %}</div>
+  </div>
+  <div id="more-infra-details" class="section section-dark">
+    <div class="container">{% include_relative _infra-more-details.html %}</div>
+  </div>
+  <div id="more-infra-details" class="section section-dark">
+    <div class="container">{% include_relative _code-viewer-css-js.html %}</div>
+  </div>
```
Comment (brikis98) [inline pages/landing/hipaa/technical-details/index.html:318]: Tacking all this stuff on to the bottom of the app page is confusing and creates a page that's large and hard to follow. Since there are two links in the nav and two links from the "how it works" section of the home page anyway, these should be two separate pages: one for reference apps, one for the reference architecture.

File: pages/landing/hipaa/technical-details/index.html [hunk 7]
```diff
@@ -53,6 +55,239 @@
...
+    image: /assets/img/hipaa/infrastructure-hipaa.png
+    image_side: right
+    body: |
+      The Reference Architecture is an opinionated, battle-tested, best-practices way to assemble the code from the <a href="/infrastructure-as-code-library"/>Infrastructure as Code Library</a>) into an end-to-end tech stack that includes just about everything you need: server cluster, load balancer, database, cache, network topology, monitoring, alerting, CI/CD, secrets management, VPN, and more (check out the <a href="/devops-checklist/">Production Readiness Checklist</a> to see what it takes to go to prod).
+
+  - heading: Customizable
+    image: /assets/img/hipaa/hipaa-sample-app.png
+    image_side: left
+    body: |
+      You can choose from EKS, ECS or EC2, databases such as MySQL, Postgres, SQL Server, to customize the Reference Architecture to your exact specifications.
```
Comment (brikis98) [inline pages/landing/hipaa/technical-details/index.html:70]: This text wasn't supposed to be used verbatim either 😕 

It needs a little copywriting to make it clearer that there are a bunch of options. And the screenshot should make the options clearer too.

File: _data/hipaa-landing-page-tabs-ref-arch-code.yml [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+        ########################################
+
+        # ---------------------------------------------------------------------------------------------------------------------
+        # CONFIGURE OUR KUBERNETES CONNECTIONS
+        # Note that we can't configure our Kubernetes connection until EKS is up and running, so we try to depend on the
+        # resource being created.
+        # ---------------------------------------------------------------------------------------------------------------------
+
+        # The provider needs to depend on the cluster being setup.
+        provider "kubernetes" {
```
Comment (brikis98) [inline _data/hipaa-landing-page-tabs-ref-arch-code.yml:18]: This code example doesn't show users what they probably look for. What is a `provider` block? How does this communicate that our code is in versioned modules? That we have out-of-the-box solutions for lots of EKS complexity?

See the [EKS example (click the tab) on the Heroku page](https://gruntwork.io/migrate-from-heroku-to-aws#step-4-deploy-to-aws) for a slightly clearer example you can use.

File: _data/hipaa-landing-page-tabs-ref-arch-code.yml [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+  content: |
+    <figure>
+      <pre>
+      <code class="language-docker">
+        #############
+        # Dockerfile
+        #############
+
+        FROM python:3.8-alpine AS with-secrets-python
+        MAINTAINER Gruntwork <info@gruntwork.io>
```
Comment (brikis98) [inline _data/hipaa-landing-page-tabs-ref-arch-code.yml:64]: We may want a Docker container for an app, not a CI / CD system here... E.g., https://github.com/gruntwork-io/aws-sample-app/blob/master/docker/Dockerfile#L53-L105

File: pages/landing/hipaa/technical-details/index.html [hunk 10]
```diff
@@ -53,6 +55,239 @@
...
+
+  - heading: Fully defined as code using Terraform, Docker, Packer
+    tab_code_preview: site.data.hipaa-landing-page-tabs-ref-arch-code
+    image_side: right
+    body: |
+      You keep and can modify 100% of the code to support any use case you might have.
+
+  - heading: Is HIPAA-compliant out of the box
```
Comment (brikis98) [inline pages/landing/hipaa/technical-details/index.html:78]:
```suggestion
  - heading: HIPAA-compliant out of the box
```

File: pages/landing/hipaa/technical-details/index.html [hunk 11]
```diff
@@ -53,6 +55,239 @@
...
+      We'll have our infrastructure and applications certified by a third party auditor to ensure HIPAA compliance, and pass all the best practices on to you.
+
+  - heading: Gruntwork deploys it into your AWS accounts and gives you all of the code to manage it, in about 1 day
+    image: /assets/img/hipaa/stay-up-to-date.png
+    image_width: 500px
+    image_side: right
+    body: |
+      Instead of spending months (or years) to re-invent the wheel, let us handle the Gruntwork for you, so you can focus on your core deliverables
+
+categories:
```
Comment (brikis98) [inline pages/landing/hipaa/technical-details/index.html:92]: This lite/pro/enterprise thing was for Heroku. HIPAA requires a different table with different features. From my original comment on this:

> Perhaps we have a similar table for HIPAA, except the columns are “Gruntwork Standard,” “Gruntwork CIS,” and “Gruntwork HIPAA” and we highlight a bunch of the HIPAA requirements as rows you only get with the HIPAA subscription: e.g., CIS hardened base images, anti-virus, file integrity monitoring, etc. We would also X out any services not eligible for HIPAA (if any).

**Review by brikis98 — COMMENTED** [review]
Thanks for putting this together Zack. However, a reminder to not copy text from GitHub/Slack comments verbatim, but to use interpretation, judgment, etc to turn those into a form a customer walking into this with 0 knowledge of Gruntwork could understand.

File: pages/landing/hipaa/technical-details/index.html [hunk 12]
```diff
@@ -8,6 +8,8 @@
     url: /hipaa-compliance-on-aws/why-gruntwork
   - name: Bootstrap your app
     url: /hipaa-compliance-on-aws/bootstrap-your-app
+  - name: HIPAA Reference Architecture
```
[the remark this answers is not in the document]
Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html]: Implemented products drop-down in daf7abe6

File: pages/landing/hipaa/technical-details/index.html [hunk 13]
```diff
@@ -8,6 +8,8 @@
     url: /hipaa-compliance-on-aws/why-gruntwork
```
[the remark this answers is not in the document]
Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html:8]: Done in 62d6f2c2

File: _data/hipaa-apps.yml (hunk 5 above)
Comment (zackproser) [reply _data/hipaa-apps.yml]: Changed to "Coming soon" for continuity with the index page's table in c1659646.

File: _data/hipaa-apps.yml (hunk 4 above)
Comment (zackproser) [reply _data/hipaa-apps.yml]: Changed to "Coming soon" for continuity with the index page's table in c165964.

File: _data/hipaa-apps.yml (hunk 2 above)
Comment (zackproser) [reply _data/hipaa-apps.yml]: Changed to "Coming soon" for continuity with the index page's table in c165964.

File: _data/hipaa-apps.yml [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+    description: Apps must look up service endpoints via environment variables, or DNS, or specific IP addresses, or a client-library.
+    available: "yes"
+
+  - title: Service mesh
+    description: After apps successfully find a service, they also need to authenticate to it, encrypt the communication channel, balance the load, and route traffic in specific ways.
+    available: "yes"
+
+  - title: Service Resilience
+    description: Apps must handle retries, exponential back-off, jitter, circuit breakers, thread management, async I/O, throttling, error handling, and a ton of other concerns.
+    available: "yes"
```
[the remark this answers is not in the document]
Comment (zackproser) [reply _data/hipaa-apps.yml]: Changed to "Coming soon" for continuity with the index page's table in c165964.

File: _data/hipaa-apps.yml (hunk 3 above)
Comment (zackproser) [reply _data/hipaa-apps.yml:16]: Updated with placeholder copy in ef2b3917.

File: pages/landing/hipaa/technical-details/index.html (hunk 7 above)
Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html]: Updated in 145e26b7

File: _data/hipaa-landing-page-tabs-ref-arch-code.yml (hunk 8 above)
Comment (zackproser) [reply _data/hipaa-landing-page-tabs-ref-arch-code.yml]: Swapped in the EKS example from the Heroku page in d31f1d14

File: _data/hipaa-landing-page-tabs-ref-arch-code.yml (hunk 9 above)
Comment (zackproser) [reply _data/hipaa-landing-page-tabs-ref-arch-code.yml]: Done in 40f48e2f

**Comment by brikis98** [comment]
Looking back at my PR comments, I came across as a bit of an asshole here. I think I had the wrong tone here and didn't capture a lot of important context. Sorry about that! I'm going to try to record a video of doing a PR review in the future and see if that provides a more effective code review, capturing my tone, adding important context, and also showing what I look for and how I think about things.

File: pages/landing/hipaa/technical-details/index.html (hunk 11 above)
Comment (ebeneliason) [reply pages/landing/hipaa/technical-details/index.html:94]: +1 on the quoted suggestion. I believe we actually had a table very much along those lines at one point, so we can probably pull it back in and add some additional rows to flesh out the distinctions more clearly.

**Comment by ebeneliason** [comment]
This page is now indeed quite long. I'd suggest one of two solutions. I like the first better, but it's more work (how much more?)
1. Add in-page tab navigation (e.g. https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior) to switch back and forth between "Reference Architecture" and "Reference Apps". This wouldn't cause a page load, but it would let users jump between them, providing a little more organization and less scrolling.
2. Add some larger headers, maybe even with some colored section dividers, to clearly separate the content into different sections on the page.

**Comment by zackproser** [comment]
> This page is now indeed quite long. I'd suggest one of two solutions. I like the first better, but it's more work (how much more?)
> 
>     1. Add in-page tab navigation (e.g. https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior) to switch back and forth between "Reference Architecture" and "Reference Apps". This wouldn't cause a page load, but it would let users jump between them, providing a little more organization and less scrolling.
> 
>     2. Add some larger headers, maybe even with some colored section dividers, to clearly separate the content into different sections on the page.

I think both approaches would work here. I can start on the tabbed navigation and report back.

File: pages/landing/hipaa/technical-details/index.html (hunk 6 above)
Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html:320]: Roger - sounds we're going to go with the in-page tabbed navigation to address this.

**Comment by zackproser** [comment]
Got the basics working in 004b1883. I think it works, but we probably want to consider some styling to make the two content tabs more obvious: 

tabbed-navigation_000

i.e., I don't want anyone missing the two navigation pills below the hero - and therefore not understanding there are two content tabs they should tab between.

@ebeneliason thoughts?

File: pages/landing/hipaa/technical-details/index.html (hunk 11 above)
Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html:94]: Roger :+1:

Comment (zackproser) [reply pages/landing/hipaa/technical-details/index.html:94]: Table implemented in 3afc7ed5. Took a first pass at the copy, but this still needs some additional HIPAA infra requirements - and some of the items that originated from the index page's table should be dropped - but the styling and structure is in place now.

**Comment by ebeneliason** [comment]
@bwhaley We'd like your input on what belongs in the table describing specific HIPAA features of the infrastructure. Here's what it looks like on the page now. The first few rows are bespoke to this table, the remainder is just copied from the home page, and might not be appropriate. Here's a relevant quote from Jim:

> Perhaps we have a similar table for HIPAA, except the columns are “Gruntwork Standard,” “Gruntwork CIS,” and “Gruntwork HIPAA” and we highlight a bunch of the HIPAA requirements as rows you only get with the HIPAA subscription: e.g., CIS hardened base images, anti-virus, file integrity monitoring, etc. We would also X out any services not eligible for HIPAA (if any).

Can you suggest anything else specific to highlight in this table?
Screen Shot 2021-07-29 at 11 36 18

**Comment by bwhaley** [comment]
- I'd leave "Log in monitoring" to be HIPAA-only
- For "Protection from malicious software", maybe we can offer automatically patched AMIs, like a nightly AMI build process. That could be HIPAA-only.
- Add a HIPAA-only row for "Inventory and categorize systems" - "Real-time reporting on cloud resources that contain PHI."
- I think the risk assessments should be reserved for HIPAA-only.
- The last two lines (Policy violations, Assign responsibility for security) should be dropped from the table.
- Anti-virus should be dropped from the table. I don't think we'll do that.

**Comment by zackproser** [comment]
>     * I'd leave "Log in monitoring" to be HIPAA-only
> 
>     * For "Protection from malicious software", maybe we can offer automatically patched AMIs, like a nightly AMI build process. That could be HIPAA-only.
> 
>     * Add a HIPAA-only row for "Inventory and categorize systems" - "Real-time reporting on cloud resources that contain PHI."
> 
>     * I think the risk assessments should be reserved for HIPAA-only.
> 
>     * The last two lines (Policy violations, Assign responsibility for security) should be dropped from the table.
> 
>     * Anti-virus should be dropped from the table. I don't think we'll do that.

Thanks, Ben. I've implemented your suggestions. Is Anti-virus something we _could_ theoretically provide for added value? Perhaps we could find a provider to partner with here?

**Comment by bwhaley** [comment]
> Thanks, Ben. I've implemented your suggestions. Is Anti-virus something we could theoretically provide for added value? Perhaps we could find a provider to partner with here?

IMO the details on how to meet this won't be fully understood until our solution is reviewed by a qualified HITRUST CSF assessor. The requirement itself comes from _§164.308(a)(5)(ii)(B) - Protection from malicious software _ which could absolutely be interpreted as A/V software.

However, HIPAA interpretations are subjective, and it's often more about meeting the spirit of the law than the letter of the law. In this case, practically speaking, does it make any sense at all to run A/V in an AWS environment? E.g. imagine that you're running an EKS cluster with tens of worker nodes, and with each node running some set of containers. Where does the A/V go? On the worker nodes? In the containers? And why? Is there a common, widespread risk of virus infection in that environment?

IMHO, the answer is no, there's practically no risk of a virus. However, there are other malicious software risks. For example, we might want a data exfiltration solution to prevent malicious software from siphoning PHI from the environment. We should also checksum and scan container images, and keep worker nodes patched. All of these could be considered "protection from malicious software" and would provide more practical value than A/V.

WDYT?

**Comment by ebeneliason** [comment]
Getting there! 
* I think it's confusing to have a products dropdown here, as this is essentially one product. Do we actually need direct links to both tabs in the nav itself?
* The links to the product pages don't work from some other pages, e.g. Why Gruntwork?
* It's weird that the link to the reference apps leads to the top of the page, but the architecture link jumps down past the hero. These should be consistent if we do link independently (though again, not sure it needs both links in the nav).
* Some of the pages still have the Technical Details link instead
* The space above the header on the two tabs differs — we should tidy that up. Relatedly, there should be more space below the header before the content starts
* I don't think the "Get a demo" button belongs here. We should replace it with a "Request early access" button. (Or cut it?)
* The selected tab style isn't quite working for me — the blue looks stronger. Maybe we need to swap the selected/unselected styles?
* Nit: it would be nice if the tabs lined up with the body content, instead of sitting all the way off to the left.
* I don't think we need the link to the comparison chart from the app details page.

**Comment by zackproser** [comment]
> I think it's confusing to have a products dropdown here, as this is essentially one product. Do we actually need direct links to both tabs in the nav itself?

Jim requested this - I do personally think it works given that we link alternately to the "apps" and the "infra" tab on the `/technical-details` page throughout the other pages. 

> The links to the product pages don't work from some other pages, e.g. Why Gruntwork?
Yep - these should all be fixed now

> Some of the pages still have the Technical Details link instead

Should be removed now!

> The space above the header on the two tabs differs — we should tidy that up. Relatedly, there should be more space below the header before the content starts

Agreed - working on it

> I don't think the "Get a demo" button belongs here. We should replace it with a "Request early access" button. (Or cut it?)

Yep - fixed!

> The selected tab style isn't quite working for me — the blue looks stronger. Maybe we need to swap the selected/unselected styles?
Nit: it would be nice if the tabs lined up with the body content, instead of sitting all the way off to the left.
I don't think we need the link to the comparison chart from the app details page.

Roger - working on it

**Comment by ebeneliason** [comment]
>Jim requested this - I do personally think it works given that we link alternately to the "apps" and the "infra" tab on the /technical-details page throughout the other pages.

Okay, in that case I think we can make it work, but we shouldn't call it "products" — I think it should still be "Technical Details" or similar.

> i.e., I don't want anyone missing the two navigation pills below the hero - and therefore not understanding there are two content tabs they should tab between.

Sorry, I missed this before. And I know you're working on the styling here already. Question: how useful is the hero in this context? Should we consider omitting it and just putting a relevant blurb under the header to add context for each page/section? I'm not sure if the tabs are more or less visible with/without it…

**Comment by zackproser** [comment]
> Question: how useful is the hero in this context? Should we consider omitting it and just putting a relevant blurb under the header to add context for each page/section? I'm not sure if the tabs are more or less visible with/without it…

They are more visible without the hero there, and a lot of what's in the hero is repeated immediately below it - so I'll try dropping it out for now.

**Comment by ebeneliason** [comment]
Okay, one more nit, and a related question…

Nit: Perhaps the Why Gruntwork? page should come next to last, before pricing, so the core details regarding what the offering _is_ come first.

Question: I know we're just using the template provided, but should we consider refactoring the way the nav works so that we don't have to modify every subpage just to change a link or swap their order? It should really be a single consistent thing across all pages, which an include is perfect for.

**Comment by ebeneliason** [comment]
I lied, more nits. :) Won't promise it's the last this time…

I'd propose dropping the "HIPAA-compliant" prefix on those nav links. I think that detail is both somewhat implied by context, and better to reiterate in the page. It makes them long and harder to distinguish between quickly.

Will "architecture" make sense as a thing for the viewer coming to this site without any knowledge of our product? Or would it be worth calling these e.g. "Infrastructure" and "Applications" (in the nav)? We could still keep the headers as they are, to introduce our spin on the solution to those two areas of interest.

**Comment by zackproser** [comment]
l

> Okay, one more nit, and a related question…
> 
> Nit: Perhaps the Why Gruntwork? page should come next to last, before pricing, so the core details regarding what the offering _is_ come first.
> 
> Question: I know we're just using the template provided, but should we consider refactoring the way the nav works so that we don't have to modify every subpage just to change a link or swap their order? It should really be a single consistent thing across all pages, which an include is perfect for.

Alright - I changed my mind about the products dropdown after implementing this. Now all nav is defined in the `landing-navbar-hipaa.html` include. Let me know what you think!

**Comment by ebeneliason** [comment]
Haha, I'm probably arguing against myself to some degree now, but as long as we're going to include both links, I actually liked them combined under one item so that we could have the more succinct, thin nav.

I was accustomed to the tab appearance before, but I think this works. The selection style is definitely more clear. I wouldn't dim the text for the unselected tab on hover — I think the background change is enough affordance. My preference would also be to left align the tabs with the page content, but maybe you tried that and it didn't work well?

**Comment by zackproser** [comment]
> Haha, I'm probably arguing against myself to some degree now, but as long as we're going to include both links, I actually liked them combined under one item so that we could have the more succinct, thin nav.
> 
> I was accustomed to the tab appearance before, but I think this works. The selection style is definitely more clear! I wouldn't dim the text for the unselected tab on hover — I think the background change is enough affordance. My preference would also be to left align the tabs with the page content, but maybe you tried that and it didn't work well?
> 
> Also, I'm still seeing inconsistent padding on those page headers. I'd add padding above the applications header to match architecture, and add a little more below both before the content.

Roger! 

- Added back in the Products dropdown in the nav
- Fixed the extraneous padding on the infra tab
- Removed the hover color change for the nav pills
- So far haven't found a good way to left-align the nav-pills with the content

**Comment by ebeneliason** [comment]
Hopefully I'm looking at the latest. I still see a few things to tweak.

* Change "Products" to e.g. "Details" or "Technical Details"
* Selecting "Reference Infrastructure" still scrolls down the page
* Padding on headers is now consistent, but I feel it should be more!

Looks great to me apart from those things. I you don't have time to address them all, I could potentially add a PR. Let's make sure we work to get what you have merged before EOD. I haven't done a full code review, but then again — do we need that level of scrutiny for this quick mockup? Probably not.

**Review by ebeneliason — APPROVED** [review]
I haven't looked closely at the code, but the preview looks good and remaining nits can be fixed in a follow up!

**Review by hposca — APPROVED** [review]
:+1:

**Comment by zackproser** [comment]
Thanks for the reviews!
